x-pack/plugin/core/src/main/java/org/elasticsearch/license/XPackLicenseState.java

```diff
@@ -507,12 +513,15 @@ public boolean isActive() {
     /**
      * Checks whether the given feature is allowed, tracking the last usage time.
      */
+    @SuppressForbidden(reason = "Argument to Math.abs() is definitely not Long.MIN_VALUE")
```

Comment:
Not necessary?  This method doesn't appear to use `Math.abs()`.

Reply:
Leftover from the previous iteration, but will come back handy with the next one )